lib/jellyfishService.js

```diff
@@ -153,60 +164,71 @@ function createServer(serverConfig, mongoClient, seagullClient, userApiClient, g
   app.use(bodyparser.json({ limit: '4mb' }));
   app.use(errorHandler({ dumpExceptions: true, showStack: true }));
 
-  app.get('/info', function(request, response) {
+  app.get('/info/?:userId?', function (request, response) {
     log.info('Handling versions request');
 
+    var uploaderDestination = 'jellyfish';
+    if (request.params.userId) {
+      log.info('using user id to detrimine upload destination');
+      if (serverConfig.uploader.platformUsers.includes(request.params.userId)) {
+        uploaderDestination = 'platform';
+        log.info(`upload to ${uploaderDestination}`);
```

Comment:
I would add the user id to the log message, too, so we can debug any problems via the logs.

Reply:
Also, I would move this outside of this if, so if there is a user id specified it logs whatever uploader destination is chosen. Much easier to investigate errors.

Reply:
So, given all of the comments in this file, something like:
```
    var uploaderDestination = null;
    if (!!request.params.userId) {
      if (serverConfig.uploader.platformUsers.includes(request.params.userId.trim().toLowerCase())) {        
        uploaderDestination = 'platform';
      } else {
        uploaderDestination = 'jellyfish';
      }
      log.info(`uploader destination for user with id '${request.params.userId}' is '${uploaderDestination}'`);
    }
```